src/pages/activitiesPage/ActivitiesPage.tsx

```diff
@@ -1,24 +1,35 @@
 ﻿import { Container } from '@mui/material';
 import { useState } from 'react';
-import Activity from '../../components/Activity';
-import AddActivity from '../../components/AddActivity';
-
-const cardStyle = {
-    width: '900px',
-    ml: 'auto',
-    mr: 'auto',
-    mb: '30px'
-}
+import ActivityButton from '../../components/ActivityButton';
+import Activity from './Activity';
+import AddActivity from './AddActivity';
+import { containerStyle, iconAlignment } from '../../muiStyles';
+import AddIcon from '@mui/icons-material/Add';
 
 const ActivitiesPage = () => {
     const [isLoggedIn, setIsLoggedIn] = useState<boolean>(true);
+    const [showAddActivity, setShowAddActivity] = useState<boolean>(false);
 
+    const handleClick = () => {
+        setShowAddActivity(current => !current)
```

Comment:
unsure about this

Reply:
Hmmm, not sure if this does what it meant to be doing.
`current => !current` is a function by itself, and it's not called.
So could it be that `setShowAddActivity(current => !current)` is setting the function as the state? 🤔 
(and not the negated value of `current` as one would expect)

Reply:
Hm, maybe, what should I use instead?
I used what was described here:
https://bobbyhadz.com/blog/react-toggle-boolean-state

Reply:
Changed it to:
`const onClick = () => {setShowAddActivity(!showAddActivity)`
instead